conda_libmamba_solver/mamba_utils.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2019 QuantStack and the Mamba contributors.
+# Copyright (C) 2022 Anaconda, Inc
```

Comment:
@jaimergp and @jezdez is it appropriate to add this line since I checked the git blame history and @jaimergp did make modifications after the QuantStack/Mamba work. I also excluded this file since it was causing precommit to fail.

Reply:
I guess it's ok :)